Documentation/Index.rst

```diff
@@ -45,3 +45,6 @@ to extend the tool.
     Developer/Index
     Migration/Index
     KnownProblems/Index
+
```

Comment:
This probably wasn't planned to commit? 😅

Reply:
woopsie